src/main/java/com/google/sps/data/EntryObject.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EntryObject {
```

Comment:
Maybe it would be good to have a more descriptive variable in this case (so others know what kind of entry you are referring to)